protocol/app/app.go

```diff
@@ -759,7 +759,6 @@ func New(
 		app.AccountKeeper,
 		app.BankKeeper,
 		app.SubaccountsKeeper,
-		memClob,
```

Comment:
note that I didn't choose to internalize this (it's still initialized in app.go) because there are a lot of tests setup that needs to be updated. so don't think it's worth it right now

Reply:
Wdym internalize? Do you think it should be initialized in the CLOB keeper constructor?

If so I don't think we should init it within the CLOB keeper constructor, since the current method of usage allows for better dependency injection.

Reply:
yeah by internalize I meant instantiate this within the clob keeper constructor. 

agree this approach allows for better dependency injection for unit tests. but for the application itself, I don't think there is a reason to expose memclob to other modules.

anyways, leaving this as is for now